Source/engine/assets.cpp

```diff
@@ -478,11 +485,24 @@ void UnloadModArchives()
 
 void LoadModArchives(std::span<const std::string_view> modnames)
 {
+	std::string targetPath;
+	for (std::string_view modname : modnames) {
+		targetPath = StrCat(paths::PrefPath(), "mods/", modname) + DirectorySeparator;
+		if (FileExists(targetPath)) {
+			OverridePaths.emplace_back(targetPath);
+		}
+		targetPath = StrCat(SDL_GetBasePath(), "mods/", modname) + DirectorySeparator;
+		if (FileExists(targetPath)) {
+			OverridePaths.emplace_back(targetPath);
+		}
+	}
+	OverridePaths.emplace_back(paths::PrefPath());
+
 #ifndef UNPACKED_MPQS
 	int priority = 10000;
-	const std::string modsPath = StrCat(paths::PrefPath(), "mods/");
+	auto paths = GetMPQSearchPaths();
 	for (std::string_view modname : modnames) {
-		LoadMPQ({ modsPath }, StrCat(modname, ".mpq"), priority);
+		LoadMPQ(paths, StrCat("mods/", modname, ".mpq"), priority);

```

Comment:
```suggestion
		LoadMPQ(paths, StrCat("mods" DIRECTORY_SEPARATOR_STR, modname, ".mpq"), priority);
```

Reply:
This results in "mod47" (probably "mod92" on Windows)

Reply:
Hm that still appears to have caused an issue

Reply:
I've commited a fix

Reply:
Thanks